src/effect.js

```diff
@@ -1,165 +1,163 @@
+ *
+ *  @method chain
+ *  @for p5.Effect
+ *  @param {Object} [arguments]  Chain together multiple sound objects
+ */
+p5.Effect.prototype.chain = function () {
+  if (arguments.length > 0) {
+    this.connect(arguments[0]);
+    for (var i = 1; i < arguments.length; i += 1) {
+      arguments[i - 1].connect(arguments[i]);
     }
-    return this._drywet.fade.value;
-  };
-
-  /**
-   *  Send output to a p5.js-sound, Web Audio Node, or use signal to
-   *  control an AudioParam
-   *
-   *  @method connect
-   *  @for p5.Effect
-   *  @param {Object} unit
-   */
-  p5.Effect.prototype.connect = function (unit) {
-    var u = unit || p5.soundOut.input;
-    this.output.connect(u.input ? u.input : u);
-  };
-
-  /**
-   * Disconnect all output.
-   * @method disconnect
-   * @for p5.Effect
-   */
-  p5.Effect.prototype.disconnect = function () {
-    if (this.output) {
-      this.output.disconnect();
-    }
-  };
-
-  p5.Effect.prototype.dispose = function () {
-    // remove refernce form soundArray
-    var index = p5sound.soundArray.indexOf(this);
-    p5sound.soundArray.splice(index, 1);
-
-    if (this.input) {
-      this.input.disconnect();
-      delete this.input;
-    }
-
-    if (this.output) {
-      this.output.disconnect();
-      delete this.output;
-    }
-
-    if (this._drywet) {
-      this._drywet.disconnect();
-      delete this._drywet;
-    }
-
-    if (this.wet) {
-      this.wet.disconnect();
-      delete this.wet;
-    }
-
-    this.ac = undefined;
-  };
-
-  return p5.Effect;
-});
+  }
+  return this;
+};
+
+/**
+ *  Adjust the dry/wet value.
+ *
+ *  @method drywet
+ *  @for p5.Effect
+ *  @param {Number} [fade] The desired drywet value (0 - 1.0)
+ */
+p5.Effect.prototype.drywet = function (fade) {
+  if (typeof fade !== 'undefined') {
+    this._drywet.fade.value = fade;
+  }
+  return this._drywet.fade.value;
+};
+
+/**
+ *  Send output to a p5.js-sound, Web Audio Node, or use signal to
+ *  control an AudioParam
+ *
+ *  @method connect
+ *  @for p5.Effect
+ *  @param {Object} unit
+ */
+p5.Effect.prototype.connect = function (unit) {
+  var u = unit || p5.soundOut.input;
+  this.output.connect(u.input ? u.input : u);
+};
+
+/**
+ * Disconnect all output.
+ * @method disconnect
+ * @for p5.Effect
+ */
+p5.Effect.prototype.disconnect = function () {
+  if (this.output) {
+    this.output.disconnect();
+  }
+};
+
+p5.Effect.prototype.dispose = function () {
+  // remove refernce form soundArray
+  var index = p5sound.soundArray.indexOf(this);
+  p5sound.soundArray.splice(index, 1);
+
+  if (this.input) {
+    this.input.disconnect();
+    delete this.input;
+  }
+
+  if (this.output) {
+    this.output.disconnect();
+    delete this.output;
+  }
+
+  if (this._drywet) {
+    this._drywet.disconnect();
+    delete this._drywet;
+  }
+
+  if (this.wet) {
+    this.wet.disconnect();
+    delete this.wet;
+  }
+
+  this.ac = undefined;
+};
+
+export default p5.Effect;
+//why can't we do export p5.Effect ?
```

Comment:
great question @endurance21 ! Without the default keyword, we'd be using a named export, and I guess "p5.Effect" is not a valid name.

We could do something like
`export const Effect = p5.Effect;` because then we'd be giving the export a valid name, which we could import as `import { Effect } from 'effect'`

and let's remove this comment from the code :)

Reply:
we can also do smth like this 
`
const Effect  = p5.Effect
export default Effect 
`

and we dont have to chnage anything in other files importing them